source/style-guide/markup/format/substitutions.txt

```diff
@@ -0,0 +1,39 @@
+=============
+Substitutions
+=============
+
+Source Constants
```

Comment:
[issue]

This seems like it needs some kind of introductory sentence or paragraph before the Source Constants heading?

Reply:
Good catch. I think it would serve the reader best to keep the title "Substitutions" since "Source Constants" might not be trivial to understand if not already familiar with it.